test/SlowTests/Issues/RavenDB-17513.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Client;
+using Raven.Client.Documents.Smuggler;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_17513 : RavenTestBase
+    {
+        public RavenDB_17513(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Theory]
+        [InlineData(1024)]
+        [InlineData(2 * 1024 + 100)]
+        [InlineData(10_000)]
+        public async Task Can_Import_Subscriptions(int count)
```

Comment:
how much time those tests are taking? if more than 1s then it should go to Stress